Rakefile

```diff
@@ -4,6 +4,9 @@
 require 'bundler'
 require 'bundler/gem_tasks'
 require 'rake/testtask'
+
+PROJECT_DIR = File.expand_path(".")
```

Comment:
I removed the references to `PROJECT_DIR` in order to reduce the coupling between rake files. Let me know what you think.